chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -412,7 +423,7 @@ impl PeerManagerActor {
                 }
             }
             // Add new edge update to the routing table and broadcast it to peers.
-            self.add_verified_edges_to_routing_table(ctx, new_edges, true);
+            self.add_verified_edges_to_routing_table(ctx, new_edges, true, false);
```

Comment:
mention why we're not updating the routing table here.

Reply:
We update routing table every 1s, so there is no need to do it immediately, after this trigger.